javalib/src/main/scala/java/lang/System.scala

```diff
@@ -23,7 +23,12 @@ object System {
   def identityHashCode(x: Object): scala.Int =
     x.cast[Word].hashCode
 
-  def getenv(name: String): String                      = ???
+  def getenv(name: String): String = {
+    val cResult = stdlib.getenv(toCString(name))
+    fromCString(cResult)
```

Comment:
I think you're correct. I've been spoiled by Rust lately and wasn't thinking about null.